src/SprykerEco/Yves/Computop/ComputopFactory.php

```diff
@@ -264,6 +292,45 @@ public function createPayPalPaymentHandler(): ComputopPrePostPaymentHandlerInter
         return new ComputopPayPalPaymentHandler($this->createInitPayPalConverter(), $this->getComputopClient());
     }
 
+    /**
+     * @return \SprykerEco\Yves\Computop\Handler\ExpressCheckout\ComputopPayPalExpressInitHandlerInterface
+     */
+    public function createPayPalExpressInitHandler(): ComputopPayPalExpressInitHandlerInterface
```

Comment:
```suggestion
    public function createComputopPayPalExpressInitHandler(): ComputopPayPalExpressInitHandlerInterface
```

Or rename the class. 

We have a convention about factory methods naming: https://spryker.atlassian.net/wiki/spaces/CORE/pages/478052552/Application+Namespace+Conventions#ApplicationNamespaceConventions-Factory
Please, check other introduced methods.